requirements.txt

```diff
@@ -6,3 +6,4 @@ nibabel>=3.0.0
 h5py>=2.8.0
 packaging>=19.0
 tqdm>=4.30.0
+trx-python==0.1
```

Comment:
why not >=0.1 ?

Reply:
I think that at least while we 'move' things it's safer to fix it until it's done.

Especially since we will be cleaning the 'core' TRX repo as we move things into Dipy.

(The goal is not to break anything, but this is only a precaution)